Resources/Prototypes/Body/Organs/vox.yml

```diff
@@ -13,3 +13,75 @@
   - type: Item
     size: Small
     heldPrefix: lungs
+
+- type: entity
+  id: OrganVoxStomach
+  parent: OrganAnimalStomach
+  name: vox stomach
+  description: "A blue stomach that smells of garbage."
```

Comment:
Idunno, maybe it's just me but I don't feel the need to plaster trash and garbage on every aspect of vox. (Additionally, I would be very surprised if it was _garbage_ that was the most prominent smell when you are handling an extracted organ oozing ammonia